File: Tugas_1/version/v.py
import sys
import platform

# Menampilkan versi interpreter Python secara detail
print(f"Versi Python: {sys.version}")

# Menampilkan informasi OS yang lebih lengkap
print(f"Sistem Operasi: {platform.platform()}")

# Menampilkan informasi engine
print(f"Engine: {platform.python_implementation()}")

# Output Contoh:
# Versi Python: 3.11.4 (main, Jun  7 2023, 10:13:09) [GCC 12.2.0]
# Sistem Operasi: Linux-6.1.0-13-amd64-x86_64-with-glibc2.36

File: Tugas_1/version/v.js
// Import modul os
const os = require('os');

// Menampilkan versi Node.js
console.log(`Versi Node.js: ${process.version}`);

// Menampilkan versi engine V8
console.log(`Versi Engine V8: ${process.versions.v8}`);

// Menampilkan platform dan rilis OS
console.log(`Sistem Operasi: ${os.platform()} ${os.release()}`);

// Output Contoh:
// Versi Node.js: v20.5.0
// Versi Engine V8: 11.3.244.8-node.17
// Sistem Operasi: darwin 22.6.0